lib/stacks/eks-blueprint-stack.ts

```diff
@@ -42,9 +42,12 @@ export class EksBlueprintProps {
     readonly version?: KubernetesVersion = KubernetesVersion.V1_20;
 
     /**
-     * VPC
+     * Named resource providers to leverage for cluster resources.
+     * The resource can represent Vpc, Hosting Zones or other resources, see {@link spi.ResourceType}.
+     * VPC for the cluster can be registed under the name of 'vpc' or as a single provider of type 
      */
-    readonly vpc?: Vpc;
+    readonly resourceProviders?: Map<string, spi.ResourceProvider> = new Map();
```

Comment:
Should we rename this to `awsResourceProviders` to make it more explicit about what resources we mean here?

Reply:
Valid point in general. I was original thinking to name it `amazonResourceProviders` similar to arn  " amazon resource notation". However, the name of the interface for the resource is `IResource` (not `IAwsResource`) so for that reason and also less verbose, I think `resourceProviders` will be consistent.